src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -228,14 +196,14 @@ def __read_features_from_file(file_path, format_):
     try:
         config_data = __read_with_appropriate_encoding(file_path, format_)
         found_feature_section = False
-        for index, feature_management_keyword in enumerate(FEATURE_MANAGEMENT_KEYWORDS):
+        for keywords in FeatureManagementReservedKeywords.ALL:
```

Comment:
nit: singular keyword?

Reply:
But then it points to multiple keywords within the single naming convention. I feel its slightly confusing.

Reply:
I think we can use singular 'keyword' here. `keyword.feature_management` still makes more sense because the different "keywords" (like feature_management) are just properties of one instance of "keyword".

Reply:
> But then it points to multiple keywords within the single naming convention. I feel its slightly confusing.

The idea was to have all the keywords corresponding to a naming convention together 
 since first we were passing `enabledFor` keyword to `__convert_feature_dict_to_keyvalue_list`.
 But we now have to pass more than just that keyword. So this was to make it easy to extend in the future without having to add another argument to the method.

Reply:
Right, and I see the source of the confusion since you are iterating through keywords, it makes semantic sense to use the singular "keyword" as the variable.

How about calling the new type `KeywordSet`? 
Then we have `for keywordset in FeatureManagementReservedKeywords.ALL`

Would this be clearer?

Reply:
that sounds good!